README.md

```diff
@@ -1 +1,177 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>To-do list</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To-do list] <a name="about-project"></a>
+
+-"To-do list" is a tool that helps to organize your day. 
+It simply lists the things that you need to do and allows you to mark them as complete.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML-5
+- CSS3
+-JAVASCRIPT
+- Linters
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.testim.io/blog/what-is-a-linter-heres-a-definition-and-quick-start-guide/">Linters</a></li>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Key features of the application.
+
+- **[input section - list section]**
+- **[Programs section - clear some or all section]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- A code editor of your choice (like vs code or atom and so on)
+-version control system (git is preferred)
+
+### Setup
+
+- Create a new folder to store the files in.
+
+### Install
+
+Install this project with:
+
+- Open your command prompt or terminal and run.
+-  git clone https://github.com/MariamOsman11/todo-list.git
+
+### Usage
+
+- use your daily task
+
+### Run tests
+
+To run tests, run the following command:
+
+- To run tests run :
+- npx hint .
+- npx stylelint "\*_/_.{css,scss}"
+
+### Deployment
+
+- Project is deployed using [GitHub Pages](https://pages.github.com/)
+- [🚀 Live Demo Link Provided Above ⬆️](#live-demo)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: https://github.com/MariamOsman
+- Twitter: https://twitter.com/@Mariannaloveee
+- [LinkedIn](https://www.linkedin.com/in/marian-isman-a57b75226/)
+
+- GitHub:https://github.com/Kanyorok
+- [LinkedIn](https://github.com/Kanyorok)
+
```

Comment:
- [ ] Here `[LinkedIn](https://www.linkedin.com/in/marian-isman-a57b75226/)` this link is giving an error and the next two links `GitHub:https://github.com/Kanyorok [LinkedIn](https://github.com/Kanyorok)` are not related to your project. Kindly this issue.